<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>379</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>379</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>379</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Procesamiento de EEG</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Titulo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>-10</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>31</red>
          <green>78</green>
          <blue>112</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>31</red>
          <green>78</green>
          <blue>112</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>SansSerif</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>ELIMINACIÃ“N DE ARTEFACTOS EN EEG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_file">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>   Examinar</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>folder.png</normaloff>folder.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_background">
    <property name="geometry">
     <rect>
      <x>-500</x>
      <y>-200</y>
      <width>1151</width>
      <height>551</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>backgroud.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_Fs">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>122</y>
      <width>159</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Frecuencia de muestreo [Hz]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_Fs">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>122</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_selectMethod">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>190</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="button_threshold">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>75</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>   Umbral</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>threshold.png</normaloff>threshold.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="button_linear">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>124</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>24</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>    Tendencia lineal </string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>linear.png</normaloff>linear.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>19</width>
      <height>19</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_kurtosis">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>220</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>    Improbabilidad</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>kurtosis.png</normaloff>kurtosis.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>22</width>
      <height>22</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_spectrum">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>79</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhUrlCharactersOnly</set>
    </property>
    <property name="text">
     <string>   Espectro</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>spectrum.png</normaloff>spectrum.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="field_fileName">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>64</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_EEG">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>64</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Archivo EEG</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_graph">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Graficar original</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_graphfilter">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Graficar filtrada</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_warningFs">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>140</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="button_guide">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>manual.png</normaloff>manual.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>label_background</zorder>
   <zorder>Titulo</zorder>
   <zorder>label_selectMethod</zorder>
   <zorder>button_file</zorder>
   <zorder>label_Fs</zorder>
   <zorder>in_Fs</zorder>
   <zorder>button_threshold</zorder>
   <zorder>button_linear</zorder>
   <zorder>button_kurtosis</zorder>
   <zorder>button_spectrum</zorder>
   <zorder>field_fileName</zorder>
   <zorder>label_EEG</zorder>
   <zorder>button_graph</zorder>
   <zorder>button_graphfilter</zorder>
   <zorder>label_warningFs</zorder>
   <zorder>button_guide</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>field_fileName</tabstop>
  <tabstop>button_file</tabstop>
  <tabstop>in_Fs</tabstop>
  <tabstop>button_graph</tabstop>
  <tabstop>button_graphfilter</tabstop>
  <tabstop>button_threshold</tabstop>
  <tabstop>button_linear</tabstop>
  <tabstop>button_kurtosis</tabstop>
  <tabstop>button_spectrum</tabstop>
  <tabstop>button_guide</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
